src/qt/transactionrecord.h

```diff
@@ -45,8 +46,12 @@ class TransactionStatus
     bool countsForBalance;
     /// Transaction was locked via InstantSend
     bool lockedByInstantSend;
+    /// Transaction was locked via ChainLocks
+    bool lockedByChainLocks;
     /// Sorting key based on status
     std::string sortKey;
+    /// Label
+    QString label;
```

Comment:
Should be initialized in ctor above

Reply:
QString has a default constructor, so there is no need to default initialize it.